container/remediate.py

```diff
@@ -27,6 +28,8 @@
 import tempfile
 import json
 import requests
+import re
+from lxml import etree
```

Comment:
lxml is extra library, we use standard elementree https://docs.python.org/2/library/xml.etree.elementtree.html

Reply:
I was afraid you say this  :grin: